### PR TITLE
[4.9] BL-9188 Fix ABC Device xmatter

### PR DESCRIPTION
**Pull request by gmartin7**
* Keep "Not For Sale" visible

---
This change is [Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/4097)

### CODE REVIEW

**Review by JohnThomson — COMMENTED** [review]
I'd like to understand this problem more deeply rather than just patch this one symptom. We should at least know, why does the page look different in Edit than when published to BR? Is Edit using Device Xmatter?
Also, is .bottomRow used for anything else, or by any other xmatter? Might we need a fix more widely?

> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/4097#-:-MN_4j1_63NrmEohkPT9:b-f4rqoj)* status: 0 of 1 files reviewed, all discussions resolved

**Review by gmartin7 — COMMENTED** [review]
I'm answering your `.bottomRow` question first:
.bottomRow is setup in `bloom-xmatter-mixins.pug` and modified by `ABC-Reader-common.pug`,  `Kyrgyzstan2020-XMatter.pug` and `Afghanistan-XMatter-mixins.pug`. In general, it's where the language name and Topic goes on the front cover.

Non-ABC books don't have a problem with BR and Edit mode difference, especially if you set the "layout" to `Device 16 x 9` before heading to `Publish -> BR`. With branding and xmatter pack set correctly for ABC, Edit mode shows ABC 1/2 Letter layout. You can switch to `Device 16 x 9` in Edit mode, and that shows the bottom scrunched like it will be without this change in BR. Unfortunately, that view doesn't use the ABC Device xmatter. Changes to that only show in BR. And because ABC Device xmatter is built off of ABC (regular) xmatter and not off of Device xmatter, it's difficult to see how to get the proper Edit mode view with the necessary bottom margin.

In conclusion, I'm not sure there's anything else to do besides what I did.

> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/4097#-:-MNdlpjXEagb98ovbwKE:b-m7t73j)* status: 0 of 1 files reviewed, all discussions resolved

**Review by JohnThomson — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/4097)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @gmartin7)
